projects/bp-gallery/src/shared/locales/de.json

```
@@ -81,7 +81,7 @@
     "wrong-time-input-info": "Jahreszahlen müssen in der Form 1989 oder 89 angegeben werden.",
     "explanation": "Jeder zusätzliche Suchbegriff schränkt die Suche weiter ein. Wenn du ohne zurückzugehen weitere\n Suchbegriffe eingibst, werden nur Bilder angezeigt, auf die alle bisherigen Suchbegriffe zutreffen.",
     "question": "Was passiert, wenn ich Stichworte suche?",
```

```suggestion
    "question": "Die Suche effektiv nutzen:",
```
I think the headline should be updated as well, shouldn't it?
(At least "Stichworte" into "Schlagworte" i would suggest)